frontend/javascripts/admin/admin_rest_api.js

```diff
@@ -1396,3 +1396,20 @@ export function computeIsosurface(
     return { buffer, neighbors };
   });
 }
+
+export function getAgglomerateSkeleton(
+  dataStoreUrl: string,
+  datasetId: APIDatasetId,
+  layerName: string,
+  mappingId: string,
+  agglomerateId: number,
+): Promise<ArrayBuffer> {
+  return doWithToken(token =>
+    Request.receiveArraybuffer(
+      `${dataStoreUrl}/data/datasets/${datasetId.owningOrganization}/${
+        datasetId.name
+      }/layers/${layerName}/agglomerates/${mappingId}/skeleton/${agglomerateId}?token=${token}`,
+      { useWebworkerForArrayBuffer: false },
```

Comment:
This is intended, I added a comment:

```
// The webworker code cannot do proper error handling and always expects an array buffer from the server.
// In this case, the server sends an error json instead of an array buffer sometimes. Therefore, don't use the webworker code.
```